src/containers/Header/index.js

```diff
@@ -13,8 +13,11 @@ import NavbarBrand from './NavbarBrand';
 export function Header(props) {
 	const HeaderService = props.app.locateService("HeaderService");
 
+	const [userbarOpen, setUserbarOpen] = useState(false);
+
 	return (
-		<header className="application-header">
+		<header className={userbarOpen ? 'op' : ''}>
```

Comment:
@petrKavulok I think here you can simplify the condition this way

```
userBarOpen && "op"
```
Also, I would use better naming - 

`userbarOpen` >>> `headerProperties`
`op` >>> `open`

or something with similar meaning